release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -682,6 +710,20 @@ $ operator-sdk new <operator_name> --type=helm \
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1874754[*BZ#1874754*])
 
+* In {product-title} 4.6, the resource metrics API server provides support for custom metrics. The resource metrics API server does not implement the OpenAPI specification, and the following messages are sent to the `kube-apiserver` logs every 60 seconds:
++
+[source,terminal]
+----
+controller.go:114] loading OpenAPI spec for "v1beta1.metrics.k8s.io" failed with: OpenAPI spec does not exist
+controller.go:127] OpenAPI AggregationController: action for item v1beta1.metrics.k8s.io: Rate Limited Requeue.
+----
++
+In some cases, these errors might cause the `KubeAPIErrorsHigh` alert to fire, but the underlying issue is not known to degrade {product-title} functionality.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1819053[*BZ#1819053*])
```

Comment:
see from https://github.com/openshift/openshift-docs/blob/10eedea2c98357298b174f297b43400c17c9dffb/release_notes/ocp-4-6-release-notes.adoc
for example, (BZ#1821771) is in separate line, do we need to do the same for (BZ#1819053) and (BZ#1870287)?

Reply:
Thank you for highlighting this. Looking at the 'Known issues' section in the Release Notes for the previous release, each of the `link` entries are included directly after the description, without a space.

Looking at the preview, this renders the BZ link at the end of the same line as the text. Looking at [the published 4.5 Release Notes Known Issues section](https://docs.openshift.com/container-platform/4.5/release_notes/ocp-4-5-release-notes.html#ocp-4-5-known-issues), this seems to be the convention. I will leave it as is based on that.